src/components/Chat/ChatItem.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from 'react'
+import * as PropTypes from 'prop-types'
+import * as cx from 'classnames'
+
+import {
+  createShorthandFactory,
+  customPropTypes,
+  IRenderResultConfig,
+  UIComponent,
+} from '../../lib'
+import { ComponentPartStyle, ComponentVariablesInput } from '../../../types/theme'
+import { Extendable, ReactChildren } from '../../../types/utils'
+import childrenExist from '../../lib/childrenExist'
+
+export interface IChatItemProps {
+  as?: any
+  content?: React.ReactNode
+  children?: ReactChildren
+  className?: string
+  styles?: ComponentPartStyle
+  variables?: ComponentVariablesInput
+}
+
+class ChatItem extends UIComponent<Extendable<IChatItemProps>, any> {
+  static className = 'ui-chat__item'
+
+  static create: Function
+
+  static displayName = 'ChatItem'
+
+  static propTypes = {
+    as: customPropTypes.as,
```

Comment:
nit: description for this prop is missing

Reply:
It's not intentional, will provide description for all prop types in the components added/changed in this PR.